kolibri/core/discovery/migrations/0009_pinned_devices_use_instance_id.py

```diff
@@ -0,0 +1,26 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.29 on 2023-03-28 17:41
+from __future__ import unicode_literals
+
+from django.conf import settings
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        migrations.swappable_dependency(settings.AUTH_USER_MODEL),
+        ("discovery", "0008_discovery_devicepinning"),
+    ]
+
+    operations = [
+        migrations.RenameField(
+            model_name="pinneddevice",
+            old_name="device_id",
+            new_name="instance_id",
+        ),
+        migrations.AlterUniqueTogether(
+            name="pinneddevice",
+            unique_together=set([("user", "instance_id")]),
+        ),
+    ]
```

Comment:
Since you have a couple intermediary migrations, it might be good to delete them and re-run `makemigrations` since I think you could collapse them all into one?